### ISSUE TITLE
[release/6.0.2xx] smoke-tests failures related to fsharp

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/source-build#2736]
Example failed CI - https://dev.azure.com/dnceng/public/_build/results?buildId=1570504&view=logs&jobId=c17db48a-1457-5682-c644-47f2df37cf6a&j=c17db48a-1457-5682-c644-47f2df37cf6a&t=32d746c7-bf36-5c0e-c923-0896ea4d665f

```
  The template "Console App" was created successfully.
      running restore
  /tarball/testing-smoke/builtCli/sdk/6.0.200/MSBuild.dll -nologo -maxcpucount -target:Restore -verbosity:m /bl:/tarball/testing-smoke/logs/FSharp_console_local_nohttps_restore.binlog ./FSharp_console.fsproj
    Determining projects to restore...
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: Unable to find package FSharp.Core with version (>= 6.0.2) [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 49 version(s) in dotnet-public [ Nearest version: 6.0.2-beta.21631.1 ] [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 1 version(s) in source-built-packages [ Nearest version: 6.0.2-beta.22057.4 ] [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in dotnet6 [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in darc-pub-dotnet-roslyn-analyzers-ce71b27 [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in darc-pub-dotnet-templating-8470ff3-1 [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in darc-pub-dotnet-templating-8470ff3-2 [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in darc-pub-dotnet-templating-8470ff3-3 [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in darc-pub-dotnet-templating-8470ff3-4 [/tarball/build.proj]
/tarball/testing-smoke/FSharp_console/FSharp_console.fsproj : error NU1102: - Found 0 version(s) in darc-pub-dotnet-templating-8470ff3 [/tarball/build.proj]
    Failed to restore /tarball/testing-smoke/FSharp_console/FSharp_console.fsproj (in 700 ms).
/tarball/build.proj(105,5): error MSB3073: The command "./smoke-test.sh --minimal --projectOutput --configuration Release --archiveRestoredPackages" exited with code 1.
```

**Comment by MichaelSimons** [comment]
[Triage] Although release/6.0.2xx is not a top priority at the moment, we do need to unblock the automation.  We should cherry-pick the testing fix for this that was used in 6.0.100

**Comment by MichaelSimons** [comment]
This no longer repos now that there was a release from 6.0.2xx.
